main.py

```diff
@@ -142,6 +153,46 @@ def create_circuit(self):
             result = 'The request body mimetype is not application/json.'
             log.debug('create_circuit result %s %s', result, 415)
             raise UnsupportedMediaType(result)
+        else:
+            log.info('############ validate spec: ################')
+            #home_dir = os.getcwd()
+            #yaml_file = "/home_dir/kytos/mef_eline/openapi.yml"
+            yaml_file = "/src/kytos-mef-eline/openapi.yml"
```

Comment:
I suggest you to use the KytosNapps api to get the napp folder and use that location as a prefix to the openai.yml file, instead of using a hardcoded value.